docs/guide-build.md

```diff
@@ -77,13 +79,13 @@ make k8s-check
 
 which will try pinging from NSCs to NSEs.
 
-You can remove the effects of k8s-deploy with:
+You can remove the effects of helm-%-install with:
```

Comment:
It is not correct command, we have format like `helm-install-%`

Reply:
thans, fixed